font/src/darwin/mod.rs

```diff
@@ -316,7 +316,14 @@ pub fn descriptors_for_family(family: &str) -> Vec<Descriptor> {
     trace!("Family: {}", family);
     let ct_collection = match create_for_family(family) {
         Some(c) => c,
-        None => return out,
+        None => {
+            // Fallback to Menlo if we can't find the config specified font family.
+            if let Some(menlo_fallback) = create_for_family("Menlo") {
+                menlo_fallback
+            } else {
+                return out;
+            }
+        }
     };
```

Comment:
I think it might be a bit nicer if we use `unwrap_or_else` here, since we don't want to change the `Some` case anyways.

Then, since this should never happen, we should probably also print a warning when the font could not be loaded using the `warn!` macro. This will lead to a warning in the Alacritty UI too, so hopefully this would make users change their config.

Reply:
Do you mean the fallback `Some` case? Or do you mean `ct_collection`'s `Some` case?

Reply:
I mean that we should do `create_from_family(family).unwrap_or_else(|| menlo.expect(...))`.

One thing I think I forgot to mention in the review is that we probably want to crash hard if menlo load isn't successful, since that's a broken system we don't need to support. Using `expect` for that should allow us to crash with a reasonable message.